### ISSUE TITLE
Editoast: MIssing details in OpenAPI for the error part

### ISSUE THREAD

**Issue by Tguisnet** [OpenRailAssociation/osrd#5706]
### Description and goal

We should add details about the error type in OpenAPI, in order to let the front-end developpers know what sort of error they could expect.

**Comment by leovalais** [comment]
Agreed on the principle, sadly rtk-query doesn't generate the types for responses other than the first 2xx. That information is still valuable as it's extra documentation, but front-end devs won't have a convenient way to interact with errors.
